src/core/store/actions/captures.js

```diff
@@ -1,4 +1,22 @@
-import { CAPTURE_CREATE, CAPTURE_DELETE } from '../../constants'
+import { pick, omitBy } from '~utils/object'
+import { CAPTURE_CREATE, CAPTURE_DELETE, SET_CAPTURE_METADATA } from '../../constants'
 
 export const createCapture = payload => ({ type: CAPTURE_CREATE, payload })
 export const deleteCapture = () => ({ type: CAPTURE_DELETE })
+
+export const setCaptureMetadata = ({ capture, apiResponse }) => {
+  const payload = {
+    captureId: capture.id,
+    metadata: omitBy(
+      {
+        ...pick(apiResponse, ["id", "side", "type"]),
```

Comment:
does `face` have a `side` in this context?

Reply:
Nope, we just pull it out as `undefined`, and then it gets taken out by the `omitBy`